app/routes/questions.$questionId.tsx

```diff
@@ -13,45 +13,43 @@ import {reloadInBackgroundIfNeeded} from '~/server-utils/kv-cache'
 const UNKNOWN_QUESTION_TITLE = 'Unknown question'
 export const LINK_WITHOUT_DETAILS_CLS = 'link-without-details'
 
+const raise500 = (error: Error) => new Response(error.toString(), {status: 500})
+
 export const loader = async ({request, params}: LoaderFunctionArgs) => {
   const {questionId} = params
   if (!questionId) {
-    throw Error('missing question title')
+    throw new Response('Missing question title', {status: 400})
   }
 
   try {
-    const dataPromise = loadQuestionDetail(request, questionId).then(({data}) => data)
-    const tagsPromise = loadTags(request).then(({data}) => data)
+    const dataPromise = loadQuestionDetail(request, questionId)
+      .then(({data}) => data)
+      .catch(raise500)
+    const tagsPromise = loadTags(request)
+      .then(({data}) => data)
+      .catch(raise500)
     return defer({data: dataPromise, tags: tagsPromise})
   } catch (error: unknown) {
-    const data: Question = {
-      pageid: questionId,
-      title: UNKNOWN_QUESTION_TITLE,
-      text: `No question found with ID ${questionId}. Please go to the Discord in the lower right (or click <a href="https://discord.com/invite/Bt8PaRTDQC">here</a>) and report where you found this link.`,
-      answerEditLink: null,
-      relatedQuestions: [],
-      tags: [],
-      banners: [],
-    }
-    return {
-      error: error?.toString(),
-      data,
-      tags: [] as Tag[],
-    }
+    const msg = `No question found with ID ${questionId}. Please go to the Discord in the lower right (or click <a href="https://discord.com/invite/Bt8PaRTDQC">here</a>) and report where you found this link.`
```

Comment:
old text that needs deleting/rewriting: `Discord in the lower right`

Reply:
I already deleted this once... :/